cou/zaza_utils/openstack.py

```diff
@@ -0,0 +1,91 @@
+import re
```

Comment:
**chore (blocking):** 

I would like to provide unit tests for the functions in this module.
It's will be nice if those string handling functions has tests.